includes/admin/extensions/product-education/class-wpsmtp.php

```diff
@@ -0,0 +1,228 @@
+<?php
+/**
+ * WP Mail SMTP
+ *
+ * Manages automatic installation/activation for WP Mail SMTP.
+ *
+ * @package     EDD
+ * @subpackage  WP_SMTP
+ * @copyright   Copyright (c) 2021, Easy Digital Downloads
+ * @license     https://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       2.11.x
+ */
+namespace EDD\Admin\Settings;
+
+class WP_SMTP {
+
+	/**
+	 * Array of configuration data for WP Mail SMTP.
+	 *
+	 * @var array
+	 */
+	private $config = array(
+		'lite_plugin'       => 'wp-mail-smtp/wp_mail_smtp.php',
+		'lite_wporg_url'    => 'https://wordpress.org/plugins/wp-mail-smtp/',
+		'lite_download_url' => 'https://downloads.wordpress.org/plugin/wp-mail-smtp.zip',
+		'pro_plugin'        => 'wp-mail-smtp-pro/wp_mail_smtp.php',
+		'smtp_settings'     => 'admin.php?page=wp-mail-smtp',
+		'smtp_wizard'       => 'admin.php?page=wp-mail-smtp-setup-wizard',
+	);
+
+	/**
+	 * The Extension Manager
+	 *
+	 * @var \EDD\Admin\Extensions\Extension_Manager
+	 */
+	private $manager;
+
+	public function __construct() {
+		add_filter( 'edd_settings_emails', array( $this, 'register_setting' ) );
+		add_action( 'edd_wpsmtp', array( $this, 'settings_field' ) );
+
+		$this->manager = new \EDD\Admin\Extensions\Extension_Manager();
+	}
+
+	/**
+	 * Register the setting to show the WP SMTP installer if it isn't active.
+	 *
+	 * @param array $settings
+	 * @return array
+	 */
+	public function register_setting( $settings ) {
+		if ( $this->is_smtp_configured() ) {
```

Comment:
Wondering if this early return should be removed, so that the SMTP field always shows. That would be consistent with Recapture. I put the early return in to be consistent with Reviews/Invoices/Recurring but this is maybe a separate scenario.

Reply:
I kind of like the early return.

Reply:
I'm okay with it too